functions/submission-created/submission-created.js

```diff
@@ -0,0 +1,39 @@
+const nodemailer = require('nodemailer')
+
+// // template inspired by https://css-tricks.com/using-netlify-forms-and-netlify-functions-to-build-an-email-sign-up-widget
+exports.handler = async event => {
+  const { EMAIL_ADDRESS, EMAIL_PASSWORD } = process.env
+  try {
+    const { name, email, message } = JSON.parse(event.body).payload.data
+
+    const transporter = nodemailer.createTransport({
+      service: 'Zoho',
+      host: 'smtp.zoho.com',
+      port: 465,
+      secure: true,
+      auth: {
+        type: 'SSL',
+        user: EMAIL_ADDRESS,
+        pass: EMAIL_PASSWORD,
+      },
+    })
+
+    const sendResult = await transporter.sendMail({
+      from: 'hello@bitsofgood.org',
+      to: 'hello@bitsofgood.org',
+      sender: `${name} <${email}>`,
```

Comment:
Actually I just did some googling, and I think this is reversed: https://stackoverflow.com/a/4367471/5233414

The Sender and Return-Path should probably be hello@bitsofgood.org while the From should be the author email.